OpenRA.Mods.Common/Traits/Crates/GrantExternalConditionCrateAction.cs

```diff
@@ -20,6 +20,9 @@ public class GrantExternalConditionCrateActionInfo : CrateActionInfo
 		[Desc("The condition to apply. Must be included in the target actor's ExternalConditions list.")]
 		public readonly string Condition = null;
 
+		[Desc("How many times to grant the condition.")]
+		public readonly int Amount = 1;
```

Comment:
I'm not convinced about calling this `Amount`. Maybe `Level` would be better? Does anyone else have an opinion here?